src/main/webapp/app/overview/course-registration-selector/course-registration-selector.component.ts

```diff
@@ -45,9 +45,11 @@ export class CourseRegistrationSelectorComponent implements OnInit {
         return new Promise<void>((resolve, reject) => {
             this.courseService.findAllToRegister().subscribe(
                 (registerRes) => {
-                    this.coursesToSelect = registerRes.body!.filter((course) => {
-                        return !this.courses.find((el) => el.id === course.id);
-                    });
+                    this.coursesToSelect = registerRes
+                        .body!.filter((course) => {
```

Comment:
I think the `.body!` part should either be on its own line or on the same line as `registerRes`

Reply:
ESLint actually does not think so, changing to proposed solutions leads to code style errors.

Reply:
Okay. It's just not the code style I'm used to.